app/assets/stylesheets/application.css.scss

```diff
@@ -107,6 +105,24 @@ td, th {
   }
 }
 
+.card .card-action a {
+  color:black;
+}
+.card .card-action a:hover {
+  color: green;
+}
+
+.btn:hover {
+  background-color: #00b8d4;
+}
+.btn, .btn-large {
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.